operator/pkg/apis/cassandra/v1beta1/webhook.go

```diff
@@ -6,26 +6,36 @@ package v1beta1
 import (
 	"errors"
 	"fmt"
-	"k8s.io/apimachinery/pkg/runtime"
 	"reflect"
+
+	"k8s.io/apimachinery/pkg/runtime"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 )
 
 var log = logf.Log.WithName("api")
 
+func attemptedTo(action string, actionStrArgs ...interface{}) error {
```

Comment:
Not sure if you guys like this type of abstraction or not.

Reply:
:+1:

Reply:
I like this, but should it be `actionStrArgs ...string` ?

Reply:
nope, because then it wouldn't support `%d` and others. I modeled it after the `fmt.Sprintf` definition

Reply:
but you always do %s below?

Reply:
oh nvm, I misread

Reply:
yeah this is a great way to keep the messaging uniform!